conn.go

```diff
@@ -263,11 +263,12 @@ func (c *Conn) handleMail(arg string) {
 			return
 		}
 	}
-	from := strings.Trim(fromArgs[0], "<> ")
+	from := strings.Trim(fromArgs[0], " ")
 	if from == "" {
 		c.WriteResponse(501, EnhancedCode{5, 5, 2}, "Was expecting MAIL arg syntax of FROM:<address>")
 		return
 	}
+	from = strings.Trim(fromArgs[0], "<>")
```

Comment:
This is more strict than before: no whitespace inside angle brackets is allowed after this commit. But maybe this is a good thing?

Reply:
I think it isnt more strict because Trim's 2nd parameter is a cutset. Before my change the whitespace was also stripped

Reply:
I mean `MAIL FROM:      <   thing >   ` would previously be parsed as `thing`, but is now parsed as `   thing `.

Reply:
Good point should bei same Logic noe